Frends.GoogleDrive.Download/Frends.GoogleDrive.Download.Test/UnitTests.cs

```diff
@@ -35,6 +35,29 @@ public void Setup()
         };
     }
 
+    [TestMethod]
+    [Ignore("Local testing only, requires shared drive setup.")]
```

Comment:
Why we can't test in on github? Can't we create some testing drive that will have some foobar file to download?
And if we want to ignore it on github, maybe it's worh to make it conditional? E.g. check if env.local exists
  [Here's](https://stackoverflow.com/questions/1594293/ignore-test-or-testfixture-based-on-a-condition) other example how we could implement conditional ignore.

Reply:
Answered in DM